docs/serving/configuration/config-defaults.md

```diff
@@ -30,319 +32,497 @@ data:
-should also be increased to prevent in-flight requests being disrupted.
+If this value is increased, the activator's `terminationGraceTimeSeconds` should also be increased to prevent in-flight requests from being disrupted.
 {% endraw %}
 
-**Key**: `max-revision-timeout-seconds`
-
-**Default**: `"600"` (10 minutes)
+* **Global key:** `max-revision-timeout-seconds`
+* **Per-revision annotation key:** `serving.knative.dev/max-revision-timeout-seconds`
+* **Possible values:** integer
+* **Default:** `"600"` (10 minutes)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  max-revision-timeout-seconds: "600" # 10 minutes
-```
 
-### Revision Cpu Request
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      max-revision-timeout-seconds: "600"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/max-revision-timeout-seconds: "600"
+        spec:
+          containers:
+            - image: gcr.io/knative-samples/helloworld-go
+    ```
+
+### Revision CPU request
 {% raw %}
-revision-cpu-request contains the cpu allocation to assign
-to revisions by default.  If omitted, no value is specified
-and the system default is used.
-Below is an example of setting revision-cpu-request.
-By default, it is not set by Knative.
+The `revision-cpu-request` value determines the CPU allocation assigned to revisions by default. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-cpu-request`
-
-**Default**: `"400m"` (0.4 of a CPU (aka 400 milli-CPU))
+* **Global key:** `revision-cpu-request`
+* **Per-revision annotation key:** `serving.knative.dev/revision-cpu-request`
+* **Possible values:** integer
+* **Default:** `"400m"` (0.4 of a CPU, or 400 milli-CPU)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-cpu-request: "400m" # 0.4 of a CPU (aka 400 milli-CPU)
-```
 
-### Revision Memory Request
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-cpu-request: "400m"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-cpu-request: "400m"
```

Comment:
This annotation doesn't exist - the config map property configures the  default value for  `Spec.Containers[*].Requests.CPU` of a `Revision`

Reply:
Can you give an example of this in the YAML or can it just not be set for individual revisions?
I don't have examples for a lot of these, so just had to work with the assumption of annotations as placeholders for documenting the revisions bit 🙂

Reply:
You would update the revision template of a Knative Service as follows:

```yaml
apiVersion: serving.knative.dev/v1
kind: Service
metadata:
  name: helloworld-go
  namespace: default
spec:
  template:
    spec:
      containers:
        - image: gcr.io/knative-samples/helloworld-go
          resources:
            requests:
              cpu: "400m"
```